app/components/dashboard/run/left-panel/component.js

```diff
@@ -110,38 +109,30 @@ export default Component.extend(FullScreenMixin, {
         this.createTooltips();
     },
 
+  
+    /**
+     * Used to check for the ?auth=true query parameter and potentially
+     * open the publish modal
+     *
+     * @method didInsertElement
+    */
     didInsertElement() {
         scheduleOnce('afterRender', this, () => {
           // Check if we're coming from an ORCID redirect
           let queryParams = this.get('params')
           if (queryParams) {
             if (queryParams.auth === 'true') {
-              this.router.transitionTo({ queryParams: { auth: null }});
-              this.send('openPublishModal', this.model.taleId);
-            }
+              this.get('store').findRecord('tale', this.model.taleId, {
+                reload: true
+              })
+              .then(resp => {
+              this.send('openPublishModal', resp)
```

Comment:
Tale should already be available via the `this.model.tale` (and will be `this.model` after the 0.8 refactoring). Was there a reason to requery for it here?